Sources/Public/MonthsLayout.swift

```diff
@@ -33,6 +33,13 @@ public enum MonthsLayout {
 
   // MARK: Internal
 
+  var isHorizontal: Bool {
```

Comment:
Feels like this kind of goes against the purpose of an enum/switch, but I guess it's just to avoid `case let` syntax.

Reply:
Yeah, it's only exposed internally but yes, it's a way to avoid an awkward case let in the `VisibleItemsProvider`.